.github/workflows/eden.yml

```diff
@@ -82,7 +82,7 @@ jobs:
             IMAGES="$IMAGES $(docker images -f reference="golang" -q)"
             IMAGES="$IMAGES $(docker images -f dangling=true -q)"
             docker rmi -f $IMAGES||echo "skip conflicts"
-            rm -rf ~/.linuxkit
+            rm -rf tmp/linuxkit-cache ~/.linuxkit
```

Comment:
You did not set env for this workflow.

Reply:
We don't need it. Isn't eden using that weird `evedanger` repo to pull it all down?

Mind, if we used cache, we could get rid of `evedanger`! OK, I will add that here.

Reply:
I mean you change directory to cleanup, but did not set env to use the directory. I think it would be better to not touch this workflow now

Reply:
Well, it cannot hurt to use the cache. Let's at least add that.

Reply:
I changed my mind. You are right. The `eden.yml` flow is different. Let's do that in a different PR.

Reply:
Will it be more clean to use env value here instead of path?

Reply:
Eventually. For now, I removed all of the eden changes.